src/wmtk/TriMeshOperationExecutor.cpp

```diff
@@ -367,6 +367,101 @@ void TriMesh::TriMeshOperationExecutor::replace_incident_face(
+
+        // do split on parent_mesh
+        Tuple ret_tuple = split_edge_single_mesh();
+
+        for (auto child_mesh_ptr : m_mesh.multi_mesh_manager.child_meshes)
+        {
+            
+            long child_id = child_mesh_ptr->multi_mesh_manager.child_id();
+            Tuple t_child = vec_t_child[child_id];
+            Tuple t_opp_child = vec_t_opp_child[child_id];
+
+            std::vector<long> child_cell_ids_to_update_hash;
+            std::vector<std::pair<long,long>> child_new_cell_ids;
+            if (child_mesh_ptr->top_simplex_type() == PrimitiveType::Face)
+            {
+                // this child_mesh is a TriMesh
+                TriMesh child_tri_mesh = *std::static_pointer_cast<TriMesh>(child_mesh_ptr);
+                if (!t_child.is_null_tuple())
+                {
+                    TriMesh::TriMeshOperationExecutor executor_child(child_tri_mesh, t_child);
+                    executor_child.split_edge();
+                    child_cell_ids_to_update_hash = executor_child.cell_ids_to_update_hash;
+                    
+                    child_new_cell_ids.push_back(std::make_pair(executor_child.m_incident_face_datas[0].split_f1, executor_child.m_incident_face_datas[0].split_f0));
+
+                    if (executor_child.m_incident_face_datas.size() > 1)
+                    {
+                        child_new_cell_ids.push_back(std::make_pair(executor_child.m_incident_face_datas[1].split_f1, executor_child.m_incident_face_datas[1].split_f0));
+                    }                    
+                }
+                if (!t_opp_child.is_null_tuple() && child_tri_mesh.simplex_is_equal(Simplex::edge(t_child), Simplex::edge(t_opp_child)))
+                {
+                    TriMesh::TriMeshOperationExecutor executor_child(child_tri_mesh, t_opp_child);
+                    executor_child.split_edge();
+                    child_cell_ids_to_update_hash.insert(child_cell_ids_to_update_hash.end(), executor_child.cell_ids_to_update_hash.begin(), executor_child.cell_ids_to_update_hash.end());
+
+                    if (child_new_cell_ids.size() == 0)
+                    {
+                        child_new_cell_ids.push_back(std::make_pair(-1, -1)); // place holder
+                    }
+                    child_new_cell_ids.push_back(std::make_pair(executor_child.m_incident_face_datas[0].split_f1, executor_child.m_incident_face_datas[0].split_f0));
+                }
+                // update_hash on new cells
+                for (long i = 0; i < long(m_incident_face_datas.size()); i++)
+                {
+                    const auto& split_fs = child_new_cell_ids[i];
+                    long split_f1 = split_fs.first;
+                    long split_f0 = split_fs.second;
+                    if (split_f1 == -1) continue;
+                    
+                    const auto& incident_face_data = m_incident_face_datas[i];
+                    long split_f0_parent = incident_face_data.split_f0;
```

Comment:
pick between whether you are going to call something child, parent, or skip the annotation (line 436 the child stuff skips being called child, but here you say parent. earlier code says parent and child. but sometimes you just say child and not parent? differnt parts of this function therefore have different variable name conventions